opm/simulators/wells/SingleWellState.hpp

```diff
@@ -85,10 +82,15 @@ class SingleWellState {
     double bhp{0};
     double thp{0};
     double temperature{0};
-    double dissolved_gas_rate{0};
-    double dissolved_gas_rate_in_water{0};
-    double vaporized_oil_rate{0};
-    double vaporized_wat_rate{0};
+
+    std::array<double,4> rates{};
```

Comment:
I'm not sure `rates` is the best name for this data member.  There are multiple other quantities in `SingleWellState` that could potentially be named `rates` as well.  As written, this is a more specialised collection that exclusively holds dissolved and vaporised flow rates and I think the name should reflect that more restricted purpose.  That said, I don't have a good suggestion for another name.  Something like `dissolved_rates` would be close, but might suggest that the vaporised components are excluded.  `Phase_mixing_rates` or just `mixing_rates` might be an alternative, but we've not used that notion anywhere else.

Reply:
This is also my only concern. Maybe `phase_mixing_rates`  will be the most descriptive. Or `mixing_rates` with some comments around can also work to me.

Reply:
let's just make it phase_mixing_rates then.